src/commands/build.ts

```diff
@@ -32,6 +32,21 @@ export class BuildCommand implements Command {
 
   args = [
     {
+      name: 'js-minify',
```

Comment:
Not so sure about this change. The reason for namespacing flags like `html.collapseWhitespace` was to at some point be able to easily map flags to build options. Continuing that pattern would lead to js.minify, css.minify and html.minify.